models/schedule_item.go

```diff
@@ -66,11 +68,101 @@ func NewTimeslotItem(t *myradio.Timeslot, finish time.Time, u func(*myradio.Time
 		Desc:    t.Description,
 		Start:   t.StartTime,
 		Finish:  finish,
-		Block:   "regular", // TODO(MattWindsor91): get this from elsewhere
+		Block:   getBock(t.Title, t.StartTime), // TODO(MattWindsor91): get this from elsewhere
 		PageURL: url.Path,
 	}, nil
 }
 
+func getBock(name string, StartTime time.Time) string{
+	name = s.ToLower(name)
+	if s.Contains(name, s.ToLower("ury: early morning")) {
+		return "flagship"
+	} else if s.Contains(name, s.ToLower("ury breakfast")) {
+		return "flagship"
+	} else if s.Contains(name, s.ToLower("ury lunch")) {
+		return "flagship"
+	} else if s.Contains(name, s.ToLower("ury brunch")) {
```

Comment:
Regardless of how above is resolved, some things here concern me:

- Renaming `strings` to `s` confuses me, because my Go naming conventions mindset's trained to expect `s` to be some throwaway local variable (like `i` or `x`, etc.);
- `s.ToLower("ury brunch")` and `s.ToLower("URY Brunch")` are the same thing, namely `"ury brunch"`;
- Running `s.ToLower` on something that's already lowercase makes no sense;
- The regularity of this block makes me feel like there should be a data structure here, eg. (excuse my half-asleep Go)

```go
type blockMatch struct {
    nameFragment string
    block string
}
var blockMatches []blockMatch // = {{ "ury: early morning", "flagship" }, ...}

for _, bm := range blockMatches {
    if s.Contains(name, s.ToLower(bm.nameFragment)) {
        return bm.block
    }
}
```

Reply:
I have implemented this method now.